### DISCUSSION TITLE
Prometheus exports for all Scans

### DISCUSSION THREAD

**Discussion by henrysachs** [aquasecurity/starboard#563] [Development]
Hi There,

is it planned to export all of the metrics from the Reports so Prometheus could scrape them? I imagined a nice Grafana Dashboard for all of these or are these Informations considered to confidential to export? Reasoning behind this is that i would like to have a way to report critical containers back to the team building them in a large cluster context where not everyone has access to the k8s cluster itself.

**Comment by jeremeb** [comment]
I used the https://github.com/kaidotdev/kube-trivy-exporter. It is working good so far.

I then used this dashboard https://grafana.com/grafana/dashboards/12331, and this one https://grafana.com/grafana/dashboards/12330.

It seems to be working good so far.

**Reply by henrysachs** [reply]
so i just had a look at it and it isn't quite what i was searching for. I was looking for an exporter of the custom ressources. Your linked exporter runs trivy scans again which is a waste of hardware ressources if I'm already doing the starboard scans isn't it?

**Comment by stone-z** [comment]
For others finding this thread, we also would like to get prometheus metrics from Starboard, so have put together [an exporter for that purpose](https://github.com/giantswarm/starboard-exporter).

There's additional discussion in [this other thread](https://github.com/aquasecurity/starboard/discussions/425) including the problem of having such high-cardinality metrics. Ours currently exposes both summary and individual CVE metrics but we're open to making that configurable if others would like to use it differently.
